conf/denas/hf/e2eaiok_denas_hf.conf

```diff
@@ -12,22 +15,28 @@ m_prob: 0.2
 s_prob: 0.4
 crossover_num: 25
 mutation_num: 25
-pretrained_model: /home/vmagent/app/dataset/bert-base-uncased
-pretrained_model_config: /home/vmagent/app/dataset/bert-base-uncased/config.json
 img_size: 128
-max_param_limits: 110
-min_param_limits: 55
 seed: 0
 expressivity_weight: 0
 complexity_weight: 0
 diversity_weight: 0.00001
 saliency_weight: 1
 latency_weight: 0.01
 
-# conf for DE-NAS BERT search space and supernet
+### Configuration for DE-NAS GPT2 search space and supernet ###
+
+## The model name used in the Hugging Face, example "prajjwal1/bert-tiny" ##
 supernet:
   bert-base-uncased
 search_space:
   intermediate_size: 
     max: 3072
-    step: 16
\ No newline at end of file
+    step: 16
```

Comment:
does that mean we still use config file instead of cmd argument to specify 'model name'

Reply:
Have refined it. Thx.